src/freenas/etc/rc.conf.local

```diff
@@ -105,6 +105,7 @@ _interface_config()
 
 		if [ "$(ha_mode)" = "MANUAL" ]; then
 			if [ "$(ulimit -n 1000; /usr/local/bin/python /usr/local/www/freenasUI/middleware/notifier.py failover_licensed 2> /dev/null)" = "True" ]; then
+				is_licensed="True"
```

Comment:
@yocalebo in this case is_licensed will only be set if ha_mode is MANUAL, which means it will never be true for our main product line.